src/frontend/src/pages/FinancePage/ReimbursementRequestForm/ReimbursementRequestForm.tsx

```diff
@@ -67,11 +67,14 @@ const schema = yup.object().shape({
     )
     .required('reimbursement products required')
     .min(1, 'At least one Reimbursement Product is required'),
-  receiptFiles: yup
-    .array()
-    .required('receipt files required')
-    .min(1, 'At least one Receipt is required')
-    .max(7, 'At most 7 Receipts are allowed')
+  receiptFiles:
+    import.meta.env.MODE === 'development'
```

Comment:
can you add a comment right above this saying what we're doing so it's obvious to anyone looking at the code

Reply:
ok also, it's a bit annoying to have someone have to go in and change code to test receipt upload. I think a better solution would be having an additional env variable in the frontend that you can set to enable receipt upload on local. Lmk if you can think of anything better, I'd rather just check if the backend env variables for receipt upload are set to check whether it should be enabled on local, but I don't think you can access them from the frontend